app/build.gradle

```diff
@@ -45,6 +45,10 @@ android {
             buildConfigField 'String', 'OFWEBSITE', '"https://world.openpetfoodfacts.org/"'
         }
     }
+    //  Crunch PNG files
+    aaptOptions {
+        cruncherEnabled = false
```

Comment:
Hi @iml-v, aapt documentation states [cruncherEnabled](https://google.github.io/android-gradle-dsl/current/com.android.build.gradle.internal.dsl.AaptOptions.html#com.android.build.gradle.internal.dsl.AaptOptions:cruncherEnabled) option is deprecated in favor of BuildType.[isCrunchPngs](https://google.github.io/android-gradle-dsl/current/com.android.build.gradle.internal.dsl.BuildType.html#com.android.build.gradle.internal.dsl.BuildType:crunchPngs)() - could you fix it please?

Reply:
@policorn thanks for suggestion , i will make changes .